tests/Dfc.CourseDirectory.Testing/TestData/TestData.CreateTLevelDefinition.cs

```diff
@@ -20,5 +22,59 @@ public async Task<Guid> CreateTLevelDefinition(Guid? tLevelDefinitionId = null,
 
             return id;
         }
+
+        public Task<IReadOnlyCollection<TLevelDefinition>> CreateInitialTLevelDefinitions() =>
```

Comment:
I think I'd rather see explicit instances being created when they're required, in a form which is suited to the test.

Reply:
That's a lot of additional ceremony for places that don't care about what the values are.

Reply:
Perhaps, but it adds isolation and stability (which is easier to factor) - I appreciate there's a balance to be had here.